x/ibc/testing/suite.go

```diff
@@ -0,0 +1,179 @@
+package testing
+
+import (
+	"time"
+
+	"github.com/stretchr/testify/suite"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	connectiontypes "github.com/cosmos/cosmos-sdk/x/ibc/03-connection/types"
+	channeltypes "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/types"
+)
+
+var (
+	ClientIDPrefix  = "clientIDForChain"
+	globalStartTime = time.Date(2020, 1, 2, 0, 0, 0, 0, time.UTC)
+	timeIncrement   = time.Second * 5
+)
+
+// IBCTestSuite is a testing struct which wraps the testing suite with N TestChain's. It handles
+// keeping all chains in time-sync.
+type IBCTestSuite struct {
```

Comment:
I'm not sure if this is the best naming. Maybe it should be renamed and just receive a `suite.T()` for testing in its constructor

Reply:
I update it to `Coordinator`, still unsure about the naming, but feels more accurate than before